### DISCUSSION TITLE
关于 local.placeholder 的语法报错，导致Hugo产生一些问题

### DISCUSSION THREAD

**Discussion by dyxang** [walinejs/waline#678] [Q&A] [answered]
在VScode种会这样显示：
![image](https://user-images.githubusercontent.com/36600710/143767658-70b641ad-c11a-4147-a806-c76aed3a97ff.png)
![image](https://user-images.githubusercontent.com/36600710/143767676-ceb8f1d4-1227-43ba-803f-542655200adf.png)

然后就导致静态博客生成器Hugo无法生成/预览/部署
```powershell
ERROR 2021/11/28 20:18:46 unexpected . in object literal on line 321 and column 132
   29:       locale.placeholder: 
```

**Comment by lizheming** [comment, marked as answer]
`local: {placeholder: '你好'}` JS 里是这么配的
